### ISSUE TITLE
Presence of decorative image and button without label in the transcription panel

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/meet#894]
## Bug Report

**Problematic behavior**
In the transcription panel, when navigating with the keyboard, "image" is read out on the button even though it has no label.

**Expected behavior/code**
Hide the image (left arrow icon)
Add a button label that clearly explains the function and action.

**Steps to Reproduce**
1. Log in
2. Open the transcription panel using keyboard navigation.
3. And then the bug happens!

**Additional context/Screenshots**
<img width="354" height="68" alt="panneau de transcription" src="https://github.com/user-attachments/assets/c57cae50-932f-4baa-beec-60d00f7fae6c" />

**Comment by Ovgodd** [comment]
Hello @cyberbaloo!
I wasn’t able to reproduce the issue on my side. The screen reader announces “Retour aux outils de réunion, button”, but it doesn’t announce “image”.
Maybe you could show me the behavior via screen sharing?

**Comment by cyberbaloo** [comment]
I just checked, it's ok! Sorry for the noise.
